lib/project_types/script/layers/infrastructure/assemblyscript_dependency_manager.rb

```diff
@@ -22,19 +22,28 @@ def installed?
         end
 
         def install
+          check_node_version!
+
           output, status = @ctx.capture2e("npm", "install", "--no-audit", "--no-optional", "--loglevel error")
           raise Errors::DependencyInstallError, output unless status.success?
         end
 
         private
 
+        def check_node_version!
+          output, status = @ctx.capture2e("node", "--version")
+          raise Errors::DependencyInstallError, output unless status.success?
+
+          version = Semantic::Version.new(output[1..-1])
+          unless version >= Semantic::Version.new("12.16.0")
+            raise Errors::DependencyInstallError, "Node version must be >= v12.16.0 Current version: #{output}."
+          end
```

Comment:
Shouldn't this string be in message.rb?

Reply:
Oops my bad, didn't know this was a thing. I'll update!